### ISSUE TITLE
Update default and minimum sizes

### ISSUE THREAD

**Issue by yexingsha** [zotero/zotero#4186]
- Window size:

	- Default (1000x600) should be larger, but I think the largest we can do is something like 1200x600, since a significant number of people are still using 1280x720 screens.
	
	- Minimum (?x300) is currently determined by item tree min width, which is 523 when both panes are closed, and 370 when either pane is open. This works fine for library view, but is too small for reader (#4185). So we need to first figure out the minimum width for reader toolbar, and then ideally determine the window min width based on the open panes both in the library and the reader, since a window that jumps in size when you switch tabs is not great.
	
- Collections pane: default/min width (200) is fine.

	- Tag selector: default height (137) can be increased to 200. And it needs a min height of 100, otherwise at smaller heights it looks broken.

- Item tree: column defaults still work great.

- Item pane: default/min width (300) can be increased to 320, so we have a little more room for the creator field mode toggle.

- Reader: other than min toolbar width to be determined in #4185, I think we want both sidebar and item pane to be open by default, so that first time users can discover them. The item pane should also default to notes.

	- Reader sidebar: default width (240) and min width (180) are both fine.

**Comment by windingwind** [comment]
> The item pane should also default to notes.

This seems to be inconsistent with the Z6's behavior?

**Comment by dstillman** [comment]
Related: https://github.com/zotero/zotero/issues/4190

**Comment by dstillman** [comment]
Yeah, I don't think defaulting to notes is right. I would guess the argument is that the notes pane encourages people to get into the recommended read-annotate-write workflow, but the item pane still feels more like the basic view that should be shown by default. That's where you'd edit or add missing metadata, maybe add tags while reading, etc., and I think it makes even more sense now that you can see everything in one view.

**Comment by dstillman** [comment]
> Default (1000x600) should be larger, but I think the largest we can do is something like 1200x600, since a significant number of people are still using 1280x720 screens.

If we can't make it taller, I think going to 1200 makes it weirdly long. 1.66 is at least pleasant.
